metal/datasource_metal_organization.go

```diff
@@ -55,6 +53,46 @@ func dataSourceMetalOrganization() *schema.Resource {
 				Computed:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			"address": {
+				Type:        schema.TypeList,
+				Description: "Business' address",
+				Computed:    true,
+				Optional:    true,
```

Comment:
`Optional: true` here (and in createOrganizationAddressDataSourceSchema) implies that address fields can be used as search filters. For the datasource, Optional should be false. 

For the resource, I think the address fields should not be computed, they should be whatever the user has defined in Terraform.

Reply:
Removed Optional from address field in DS schema. Same for address sub-schema fields. Resource was fine without compute, nevertheless I have removed those occurrences of required=false and optional=false to make it cleaner